.github/actions/integration-tests/action.yml

```diff
@@ -33,6 +33,10 @@ inputs:
     description: "Which rickshaw endpoint to exercise"
     required: false
     default: "remotehost"
+  ci_build_controller:
+    description: "Should the crucible controller be rebuilt and tested"
+    required: false
+    default: "no"
```

Comment:
here default=no

Reply:
I'm not sure I follow what you mean here, can you elaborate?